app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -267,6 +267,22 @@
         )
     },
     classes = "govuk-input--width-20"))
+@govukInput(Input(
+id = FieldNames.formerReferencePro,
+name = FieldNames.formerReferencePro,
+value = editSetRecordForm.data.get(FieldNames.formerReferencePro),
+label = Label(
+forAttr = Some(FieldNames.formerReferencePro),
+content = Text(messages("edit-set.record.edit.former-reference-pro")),
+classes = "govuk-label--s"
+),
+errorMessage = editSetRecordForm.error(FieldNames.formerReferencePro).map { error =>
+ErrorMessage(
+content = Text(error.message),
+attributes = Map("for" -> FieldNames.formerReferencePro)
+)
+},
+classes = "govuk-input--width-20"))
```

Comment:
Can we indent this section, as we would with normal Scala code, so it is a bit easier to read please.

Reply:
Done